### PR TITLE
Fix spelling issues

### PR DESCRIPTION
**Pull request by szemere**
Reported by: #3394 there were several typing mistakes (mostly in the debug messages).

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Review by bazsi — COMMENTED** [review]
This looks good to me. I think there are only few patches flowing into db-parser() at this point, so renaming stuff is possible without causing too much headaches (ie. conflicts).

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Review by furiel — DISMISSED** [review]

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: news/bugfix-3398.md [hunk 1]
```diff
@@ -0,0 +1 @@
+`syslog-ng`: fix spelling issues
```
Comment (bazsi) [inline news/bugfix-3398.md:1]: I would extend this like: "fixed numerous spelling mistakes in messages generated by syslog-ng"

**Comment by bazsi** [comment]
Apart from a bit more elaboration in the news file, this looks good to me.

**Review by bazsi — APPROVED** [review]

**Review by MrAnno — APPROVED** [review]

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Review by MrAnno — APPROVED** [review]

**Review by bazsi — APPROVED** [review]

**Pull request merged**